tests/system/test_system.py

```diff
@@ -102,6 +102,20 @@ def test_allocate_ids(self):
         self.assertEqual(len(unique_ids), num_ids)
 
 
+class TestDatastoreReserveIDs(TestDatastore):
+    def test_reserve_ids_sequential(self):
+        # Smoke test to make sure it doesn't blow up. No return value or
+        # verifiable side effect to verify.
+        num_ids = 10
+        Config.CLIENT.reserve_ids_sequential(Config.CLIENT.key("Kind", 1234), num_ids)
+
+    def test_reserve_ids_multi(self):
+        # Smoke test to make sure it doesn't blow up. No return value or
+        # verifiable side effect to verify.
+        keys = [Config.CLIENT.key("KIND", 1234), Config.CLIENT.key("KIND", 1235)]
+        Config.CLIENT.reserve_ids_multi(keys)
```

Comment:
Prolly needs a test for `reserve_ids`, including a `with warnings.catch_warnings(log=True) as warned:` wrapper (assuming  you agree with me above).

Reply:
d03607c